beacon_node/beacon_chain/src/beacon_chain.rs

```diff
@@ -4808,7 +4893,9 @@ impl<T: BeaconChainTypes> BeaconChain<T> {
 
         // If we are close enough to the proposal slot, send an fcU, which will have payload
         // attributes filled in by the execution layer cache we just primed.
-        if till_prepare_slot <= self.config.prepare_payload_lookahead {
+        if self.config.always_prepare_payload
```

Comment:
I'm wondering why we want to start updating the EE earlier when we have `always_prepare_payload == true`? I don't think it's necessarily bad, just wondering.

Reply:
It helps in the case of a skipped slot or 1-slot re-org where the builder wants to potentially start building a payload on the parent block as soon as the slot begins. I.e. build a slot `n + 1` block on the block from slot `n - 1` at the start of slot `n`.

Reply:
I guess it's a bit redundant though as builders are probably setting the payload lookahead to 12s anyway

Reply:
Seems reasonable to me! It's an opt-in flag so I can't see any change in behavior here being a serious issue!